riscv/insns/vcompress_vm.h

```diff
@@ -1,14 +1,14 @@
 // vcompress vd, vs2, vs1
+P.any_vector_extensions();
```

Comment:
Looks like a statement with no effect; do you mean `require(P.any_vector_extensions());`?

Reply:
@aswaterman
Yes, we need the `require()`. Thank you for pointing out.